code/datums/weather/weather_controller.dm

```diff
@@ -87,8 +87,8 @@
 			/datum/weather/hailstorm = 20
 			)
 
-/datum/weather_controller/desert
-	possible_weathers = list(/datum/weather/sandstorm = 100)
+/datum/weather_controller/wasteland
+	possible_weathers = list(/datum/weather/sandstorm = 30, /datum/weather/acid_rain = 60, /datum/weather/shroud_storm = 10)
```

Comment:
This isn't modularized, but im pretty sure weather controllers aren't being carried over with the rebase, since we're getting actual weather

Reply:
What do you mean actual weather

Reply:
zlevels use a different system entirely, weather_controllers are a virtual zlevel thing

Reply:
we'll have to address that once we get to it, we don't want to lose our weather functionality